### PR TITLE
Add support for passing resource limits to ImageMagick 'convert'

### PR DESCRIPTION
**Pull request by bladealslayer**

### CODE REVIEW

File: config/config.defaults.yml [hunk 1]
```diff
@@ -253,6 +253,18 @@ default: &default_settings
   # Max filesize for uploaded image (bytes)
   max_image_filesize: 104857600
 
+  # Memory limit for image processing - passed to ImageMagick -limit memory (MAGICK_MEMORY_LIMIT)
+  # See http://www.imagemagick.org/script/resources.php
+  # Example value:
+  # image_processing_memory_limit: 512MiB
+  image_processing_memory_limit:
```
Comment (rap1ds) [inline config/config.defaults.yml]: Could we add here a sensible default value?

File: config/config.defaults.yml [hunk 2]
```diff
@@ -253,6 +253,18 @@ default: &default_settings
...
+
+  # Memory map limit for image processing - passed to ImageMagick -limit map (MAGICK_MAP_LIMIT)
+  # See http://www.imagemagick.org/script/resources.php
+  # Example value:
+  # image_processing_map_limit: 512MiB
+  image_processing_map_limit:
```
Comment (rap1ds) [inline config/config.defaults.yml]: Same here

File: config/config.defaults.yml (hunk 1 above)
Comment (bladealslayer) [inline config/config.defaults.yml]: I had trouble deciding what the "sensible" defaults would be so left the default behaviour unchanged. What do you think?

Comment (rap1ds) [inline config/config.defaults.yml]: Understood. Leave it empty then.

**Pull request merged**